alivee/place_search_engine/constants/google_place_api.py

```diff
@@ -0,0 +1,255 @@
+	PlaceServicePlaceType.CAR_DEALER,
+	PlaceServicePlaceType.CAR_RENTAL,
+	PlaceServicePlaceType.CAR_REPAIR,
+	PlaceServicePlaceType.CAR_WASH,
+	PlaceServicePlaceType.CASINO,
+	PlaceServicePlaceType.CEMETERY,
+	PlaceServicePlaceType.CHURCH,
+	PlaceServicePlaceType.CITY_HALL,
+	PlaceServicePlaceType.CLOTHING_STORE,
+	PlaceServicePlaceType.CONVENIENCE_STORE,
+	PlaceServicePlaceType.COURTHOUSE,
+	PlaceServicePlaceType.DENTIST,
+	PlaceServicePlaceType.DEPARTMENT_STORE,
+	PlaceServicePlaceType.DOCTOR,
+	PlaceServicePlaceType.ELECTRICIAN,
+	PlaceServicePlaceType.ELECTRONICS_STORE,
+	PlaceServicePlaceType.EMBASSY,
+	PlaceServicePlaceType.FIRE_STATION,
+	PlaceServicePlaceType.FLORIST,
+	PlaceServicePlaceType.FUNERAL_HOME,
+	PlaceServicePlaceType.FURNITURE_STORE,
+	PlaceServicePlaceType.GAS_STATION,
+	PlaceServicePlaceType.GYM,
+	PlaceServicePlaceType.HAIR_CARE,
+	PlaceServicePlaceType.HARDWARE_STORE,
+	PlaceServicePlaceType.HINDU_TEMPLE,
+	PlaceServicePlaceType.HOME_GOODS_STORE,
+	PlaceServicePlaceType.HOSPITAL,
+	PlaceServicePlaceType.INSURANCE_AGENCY,
+	PlaceServicePlaceType.JEWELRY_STORE,
+	PlaceServicePlaceType.LAUNDRY,
+	PlaceServicePlaceType.LAWYER,
+	PlaceServicePlaceType.LIBRARY,
+	PlaceServicePlaceType.LIQUOR_STORE,
+	PlaceServicePlaceType.LOCAL_GOVERNMENT_OFFICE,
+	PlaceServicePlaceType.LOCKSMITH,
+	PlaceServicePlaceType.LODGING,
+	PlaceServicePlaceType.MEAL_DELIVERY,
+	PlaceServicePlaceType.MEAL_TAKEAWAY,
+	PlaceServicePlaceType.MOSQUE,
+	PlaceServicePlaceType.MOVIE_RENTAL,
+	PlaceServicePlaceType.MOVIE_THEATER,
+	PlaceServicePlaceType.MOVING_COMPANY,
+	PlaceServicePlaceType.MUSEUM,
+	PlaceServicePlaceType.NIGHT_CLUB,
+	PlaceServicePlaceType.PAINTER,
+	PlaceServicePlaceType.PARK,
+	PlaceServicePlaceType.PARKING,
+	PlaceServicePlaceType.PET_STORE,
+	PlaceServicePlaceType.PHARMACY,
+	PlaceServicePlaceType.PHYSIOTHERAPIST,
+	PlaceServicePlaceType.PLUMBER,
+	PlaceServicePlaceType.POLICE,
+	PlaceServicePlaceType.POST_OFFICE,
+	PlaceServicePlaceType.REAL_ESTATE_AGENCY,
+	PlaceServicePlaceType.RESTAURANT,
+	PlaceServicePlaceType.ROOFING_CONTRACTOR,
+	PlaceServicePlaceType.RV_PARK,
+	PlaceServicePlaceType.SCHOOL,
+	PlaceServicePlaceType.SHOE_STORE,
+	PlaceServicePlaceType.SHOPPING_MALL,
+	PlaceServicePlaceType.SPA,
+	PlaceServicePlaceType.STADIUM,
+	PlaceServicePlaceType.STORAGE,
+	PlaceServicePlaceType.STORE,
+	PlaceServicePlaceType.SUBWAY_STATION,
+	PlaceServicePlaceType.SYNAGOGUE,
+	PlaceServicePlaceType.TAXI_STAND,
+	PlaceServicePlaceType.TRAIN_STATION,
+	PlaceServicePlaceType.TRANSIT_STATION,
+	PlaceServicePlaceType.TRAVEL_AGENCY,
+	PlaceServicePlaceType.UNIVERSITY,
+	PlaceServicePlaceType.VETERINARY_CARE,
+	PlaceServicePlaceType.ZOO,
+]
+
+
+# Place types could be retured in place search response.
+# Documents: https://developers.google.com/places/web-service/supported_types#table2
+PLACE_SERVICE_RESPONSE_PLACE_TYPES = [placeType 
+	for placeType in PlaceServicePlaceType.__dict__.keys() if not placeType.startswith('__')]
```

Comment:
Let's consider using [enum](https://docs.python.org/3/library/enum.html) for these constants. It has interface we can use here to list all the type members.